templates/badges.jade

```diff
@@ -27,6 +27,10 @@ mixin badge-coverage(repo)
   a(href='https://coveralls.io/r/' + repo)
     +badge('/coveralls/c/github/' + repo + '/master')
 
+mixin badge-packagephobia(npm)
+  a(href='https://https://packagephobia.now.sh/result?p=' + npm)
```

Comment:
Double `https://` ☺️

Reply:
@LinusU Nice catch! Copy/paste fail 😄
I fixed it!